.ci/manywheel/build_cuda.sh

```diff
@@ -43,13 +43,6 @@ if [[ -n "$DESIRED_CUDA" ]]; then
         fi
     fi
     echo "Using CUDA $CUDA_VERSION as determined by DESIRED_CUDA"
-
-    # There really has to be a better way to do this - eli
-    # Possibly limiting builds to specific cuda versions be delimiting images would be a choice
-    if [[ "$OS_NAME" == *"Ubuntu"* ]]; then
-        echo "Switching to CUDA version ${DESIRED_CUDA}"
-        /builder/conda/switch_cuda_version.sh "${DESIRED_CUDA}"
```

Comment:
I believe this code is not used for a while The PR to remove switch_cuda_versions.sh was merged on November 19 https://github.com/pytorch/builder/pull/2041